pkgs/games/tetrio-desktop/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, alsa-lib
+, cups
+, libX11
+, libXScrnSaver
+, libXtst
+, mesa
+, nss
+, systemd
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tetrio-desktop";
+  version = "8.0.0";
+
+  src = fetchurl {
+    url = "https://tetr.io/about/desktop/builds/TETR.IO%20Setup.deb";
+    name = "${pname}.deb";
+    sha256 = "1nlblfhrph4cw8rpic9icrs78mzrxyskl7ggyy2i8bk9i07i21xf";
+  };
+
+  nativeBuildInputs = [
+    alsa-lib
+    autoPatchelfHook
+    cups
+    libX11
+    libXScrnSaver
+    libXtst
+    mesa
+    nss
+    wrapGAppsHook
```

Comment:
Why do we need those libraries in here?

Reply:
if I remove them, it fails to build or immediately crashes when I run it

Reply:
I'll check

Reply:
`autoPatchelfHook` needs them